FWCore/MessageService/test/filter-timestamps.sed

```diff
@@ -1 +1 @@
-s/ [0-9]?[0-9]-(Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec)-2[0-9][0-9][0-9] [0-9]?[0-9]:[0-9][0-9]:[0-9][0-9](\.[0-9][0-9][0-9])? [A-Z]?[A-Z][A-Z][A-Z]( |\t|$)/ {Timestamp} /
+s/ [0-9]?[0-9]-(Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec)-2[0-9][0-9][0-9] [0-9]?[0-9]:[0-9][0-9]:[0-9][0-9](\.[0-9][0-9][0-9])? [A-Z]?[A-Z][A-Z][A-Z]( |\t|-|$)/ {Timestamp} /
```

Comment:
One of the unit tests was failing because of an extra minus sign that happened in the time printout at its end. I can't remember exactly which one. I looked into and determined the code was doing the right thing and the filtering was just missing a corner case.